src/main/resources/db/migration/local/V1_243_2__add_external_test_security_groups.sql

```diff
@@ -0,0 +1,49 @@
+UPDATE darts.user_account
+SET account_guid = '78567440-d51c-4665-8664-218bd578dc68', is_system_user = true, user_name = 'cpp_user'
+WHERE usr_id = -41;
+
+UPDATE darts.user_account
+SET account_guid = 'd040e413-f1ac-4902-b347-369f6e582559', is_system_user = true, user_name = 'xhibit_user'
+WHERE usr_id = -40;
+
+UPDATE darts.security_group
+SET global_access = true
+WHERE group_name in ('Dar Pc Group', 'Mid Tier Group');
+
+INSERT INTO darts.security_group_user_account_ae
+(usr_id, grp_id)
+VALUES(-46, -17);
+INSERT INTO darts.security_group_user_account_ae
+(usr_id, grp_id)
+VALUES(-46, -16);
+INSERT INTO darts.security_group_user_account_ae
+(usr_id, grp_id)
+VALUES(-46, -15);
+
+
+INSERT INTO darts.user_account
+(usr_id, dm_user_s_object_id, user_name, user_email_address, description, user_state, created_ts, last_modified_ts, last_login_ts, last_modified_by, created_by, account_guid, is_system_user)
+VALUES(-60, NULL, 'darpc_midtier_user', 'darpc_midtier_user@hmcts.net', NULL, NULL, NULL, NULL, NULL, NULL, NULL, 'b884e22d-a38c-4a34-94d0-af7f0e177bc6', true);
+
+INSERT INTO darts.security_group_user_account_ae
+(usr_id, grp_id)
+VALUES(-60, -17);
+
+INSERT INTO darts.security_group_user_account_ae
+(usr_id, grp_id)
+VALUES(-60, -16);
+
+UPDATE darts.user_account
+SET account_guid = '9797a4fc-8e62-4095-bda6-04e7747b05e5'
+WHERE usr_id = -46;
+
+-- due to a clash of user accounts whereby there are now 2 darts_global_test_users, rename the one on test -44 and the one on staging -48 N.B. can't remove users due to foreign keys
```

Comment:
@hemantasharma1129 / @karen-hedges I don't understand why there are 2 users tbh. Would we be expecting to set this global user for testing purposes to be an ADMIN group. I did this locally to be able to use the new user search endpoint: /users/search

```
-- darts.global.user@hmcts.net
INSERT INTO security_group_user_account_ae (usr_id, grp_id)
VALUES (-46, 1);
```

Reply:
Locally, I appear to have usr_id: 44, 46 as darts.global.user@hmcts.net?

Reply:
Local has 2 -44 and -46 and staging has 2 -46 and -48. This was because I added the first global user with -44 but only checked localhost id's instead of staging whereby someone else had created a user with that same id.  The OID I set was for the old azure settings. However, Hemanta was changing the azure stuff and consequently he created a new global user with the new azure OID. This then allowed testers to use both users. -46 appears on both DB's so it makes sense to make that one the main global user.